content/en/synthetics/guide/reusing-browser-test-journeys.md

```diff
@@ -0,0 +1,73 @@
+
+To create a login test and use it as a subtest in the rest of your test suite:
+
+1. Create a first test A that does nothing but logging into your application. You can set the **start URL** of this test A to your pre login URL.
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/login_subtest_recording.mp4" alt="Recording the Login subtest" video="true"  width="100%">}}
+
+2. Create a second test B that monitors any post login functionalities of your application. You can set the **start URL** of this test B to your pre login URL as well.
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/dashboard_test_configuration.png" alt="Configuring the parent test" >}}
+
+In the above example, you can see that this second test focuses on dashboard creation monitoring.
+
+3. Once on the recorder of your test B, click the Subtest button and select the login test A you just created. 
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/dashboard_test_subtest.mp4" alt="Including subtest in parent test" video="true"  width="100%">}}
+
+You just created a **Subtest** step: all the steps of the subtest (ie. test A) are consequently played at the beginning of your parent test (ie. test B).
+By default, the subtest is played in the main tab. This means your subtest steps are played in the same tab as the previous and following steps. The subtest starts running using the URL that was set in the parent test (ie. the pre login URL), and once all the subtest steps have been executed, the Browser test executes the parent’s first non subtest step from the page the subtest was last on. We did not set any parent step for now.
+
+You can only see that after importing your login subtest (ie. test A) in your parent test (ie. test B), the two variables from the login subtest are imported into your parent test as well.
+
+**Note:** You can choose the tab in which the subtest should execute using [**Subtest Advanced Options**][1].
+
+4. Login to your account with the dedicated credentials in the iframe/pop up associated with the recorder in order to start recording your parent test’s steps from the same state your Browser test would be in after going through the subtest’s steps.
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/dashboard_test_iframe.mp4" alt="Replaying subtest in parent test" video="true"  width="100%">}}
+
+5. Hit **Start recording** once you’re logged in to start recording the parent test’s post login steps you’re interested in. Once you’re done hit **Save**.
+
+{{< img src="synthetics/guide/reusing-browser-test-journeys/dashboard_test_recording.mp4" alt="Recording parent test" video="true"  width="100%">}}
+
+In the above example we are checking that after logging into a test Datadog account, users are able to create a timeboard and that the timeboard is associated with the user who created it.
```

Comment:
This might be a big request, but testing Datadog with Datadog could be a bit confusing to a reader, as it's blurry where we're showing the product vs. where we're showing testing the product. Creating a test on a sample application would eliminate this confusion.

Reply:
That's a big request indeed :) but I fully agree with you. I'll DM you to discuss that!